pandas/tests/test_accent.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+from pandas import read_csv, DataFrame
+from pandas.util.testing import ensure_clean
+
+class FileSystemEncodingTest(unittest.TestCase):
+    """Test compatibility with file system encoding"""
+    def test_load(self):
+        for filename in ('test_e.txt', 'test_é.txt'):
+            with ensure_clean(filename) as path:
```

Comment:
The problem is with Unicode paths, not byte path.

I used `os.path.join(pandas.util.testing.get_data_path(), 'test_é.txt')` in the previous test version which is also a Unicode path and failed as intended with AppVeyor.